src/unity/toolkits/activity_classification/activity_classifier.cpp

```diff
@@ -136,6 +138,12 @@ std::tuple<gl_sframe, gl_sframe> random_split_by_session(gl_sframe data, std::st
 
 }  // namespace
 
+struct result {
```

Comment:
Let's define this new type inside the anonymous namespace, to guard against future name collisions